api-specificatie/openapi.yaml

```diff
@@ -1306,67 +1272,60 @@ components:
       properties:
         woonplaats:
           $ref: '#/components/schemas/WoonplaatsHalBasis'
-          description: 'De aan het oenbareruimte gerelateerde woonplaats (object).'
+          description: "De aan het oenbareruimte gerelateerde woonplaats (object)."
 
     Nummeraanduiding:
       type: object
-      title: Nummeraanduiding
-      description: 'Een Nummeraanduiding is een door het bevoegde gemeentelijke orgaan als zodanig toegekende aanduiding van een verblijfsobject, een standplaats of een ligplaats. Deze definitie is herleidbaar naar maar niet conform de BAG Catalogus 2018.'
+      description: "Een postcode, huisnummer met mogelijk een huisletter en -toevoeging die hoort bij een verblijfsobject, een standplaats of een ligplaats."
       properties:
         identificatie:
           $ref: '#/components/schemas/NEN3610ID'
         huisnummer:
-          description: Een door of namens het gemeentebestuur ten aanzien van een adresseerbaar object toegekende nummering.
-          title: huisnummer
+          description: "Nummer dat door de gemeente aan een adresseerbaar object is gegeven."
           type: integer
           example: '12'
         huisletter:
-          description: Een door of namens het gemeentebestuur ten aanzien van een adresseerbaar object toegekende toevoeging aan een huisnummer in de vorm van een alfanumeriek teken.
-          title: huisletter
+          description: "Toevoeging aan een huisnummer in de vorm van een letter of een cijfer die door de gemeente aan een adresseerbaar object is gegeven."
           type: string
           example: 'A'
         huisnummertoevoeging:
-          description: Een door of namens het gemeentebestuur ten aanzien van een adresseerbaar object toegekende nadere toevoeging aan een huisnummer of een combinatie van huisnummer en huisletter.
-          title: huisnummer toevoeging
+          description: "Toevoeging aan een huisnummer of een combinatie van huisnummer en huisletter die door de gemeente aan een adresseerbaar object is gegeven."
           type: string
           example: 'bis'
         postcode:
-          title: postcode
-          description: De door PostNL vastgestelde code behorende bij een bepaalde combinatie van een straatnaam en een huisnummer.
+          description: "Door PostNL vastgestelde code die bij een bepaalde combinatie van een straatnaam en een huisnummer hoort."
           type: string
           example: '7500AA'
         status:
           $ref: '#/components/schemas/StatusNaamgeving_enum'
         geconstateerd:
-          description: "Indicator dat een object in de registratie is opgenomen op basis van een feitelijke constatering, zonder dat er een besluit en brondocument aan ten grondslag ligt."
+          description: "Indicatie dat de nummeraanduiding in de registratie is opgenomen door een feitelijke constatering, zonder dat er sprake was van een brondocument/vergunning. De nummeraanduiding is mogelijk illegaal."
```

Comment:
waarom heb je van Indicator Indicatie gemaakt?
Voor mij is een indicatie een aanwijzing dat iets misschien zo is, een indicator is een aanduiding dat iets zo is.

Reply:
Ik neem aan dat je vraag aan mij gericht is.... Cathy heeft deze teksten in #151 aangeleverd. Ik heb die teksten niet allemaal inhoudelijk gereviewed. Ik heb hier dus geen motivatie voor. 

Ik ga je feedback verwerken omdat op alle punten de descriptions er beter van worden. Mocht @CathyDingemanse er anders over denken dan is dat altijd weer aanpasbaar.